tests/eth2/fixtures-tests/state-fixtures/test_minimal_state.py

```diff
@@ -157,7 +157,7 @@ def execute_state_transtion(test_case, base_db):
 
     post_state = pre_state.copy()
     for block in blocks:
-        sm = sm_class(chaindb, attestation_pool, None, post_state)
+        sm = sm_class(chaindb, attestation_pool, None, state=post_state)
```

Comment:
Would it be more readable to pass all of them in kwargs?
```suggestion
        sm = sm_class(
            chaindb=chaindb,
            attestation_pool=attestation_pool,
            slot=None,
            block=None,
            state=post_state,
        )
```